plugins/kotlin/code-insight/impl-base/src/org/jetbrains/kotlin/idea/codeinsights/impl/base/inspections/ReplaceGetOrSetInspectionUtils.kt

```diff
@@ -10,9 +10,9 @@ import org.jetbrains.kotlin.util.OperatorNameConventions
 
 object ReplaceGetOrSetInspectionUtils {
     private val operatorNames: Set<Name>
-        get() = setOf(OperatorNameConventions.GET, OperatorNameConventions.SET)
```

Comment:
The code of inspection should not be changed, we should just not apply it in a post-processing.